core/src/test/scala/neotypes/ConcurrentSessionSpec.scala

```diff
@@ -0,0 +1,38 @@
+package neotypes
+
+import neotypes.implicits.mappers.all._
+import neotypes.implicits.syntax.cypher._
+import neotypes.internal.syntax.async._
+import org.neo4j.driver.v1.exceptions.ClientException
+import org.scalatest.compatible.Assertion
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+
```

Comment:
Intellij tells me that you can clean up the imports.

Reply:
Will take a look :+1: